usmqe_tests/web/tendrl/conftest.py

```diff
@@ -29,3 +30,26 @@ def log_in(testcase_set):
         "should contain all required components."
     pytest.check(testcase_set.init_object.is_present, msg)
     yield testcase_set
+
+
+@pytest.fixture(scope="function")
+def log_out(testcase_end):
+    """ fixture for logging out """
+    yield testcase_end
+    upper_menu = UpperMenu(testcase_end.driver)
+    user_menu = upper_menu.open_user_menu()
+    user_menu.logout()
+
+
+@pytest.fixture(scope="function")
+def valid_credentials(log_in, log_out):
```

Comment:
I like the idea about using log_out fixture but it is not explicitly clear at the first sight. Maybe some comment about usage would be good.